DigitalLearningSolutions.Data/Models/CustomPrompts/Prompt.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class Prompt
+    {
+        public string PromptText { get; set; }
+        public List<string> Options { get; set; }

```

Comment:
Let's set these in a Prompt constructor rather than having to do it in every class that extends Prompt.

Reply:
done